bundles/org.eclipse.equinox.security/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %pluginName
 Bundle-SymbolicName: org.eclipse.equinox.security;singleton:=true
-Bundle-Version: 1.4.300.qualifier
+Bundle-Version: 1.4.301.qualifier
```

Comment:
We generally increment by +100 in the master stream so that there is room for increments in a maintenance stream that don't overlap with the  master stream's version.

https://github.com/eclipse-platform/eclipse.platform/blob/master/docs/VersionNumbering.md

Reply:
Thanks for the pointer Ed.